Description: Skip online tests
 Some tests rely on access to online resources; skip
 these as this does not work in offline build environments.
Author: James Page <james.page@ubuntu.com>
Forwarded: no

--- a/go/vcs/vcs_test.go
+++ b/go/vcs/vcs_test.go
@@ -43,6 +43,7 @@ func TestRepoRootForImportPath(t *testin
 
 	for _, test := range tests {
 		got, err := RepoRootForImportPath(test.path, false)
+		t.Skip("Skipping test in offline build environment")
 		if err != nil {
 			t.Errorf("RepoRootForImport(%q): %v", test.path, err)
 			continue
